env/vars.go

```diff
@@ -3,22 +3,25 @@ package env
-	ScopeTestingPanicAsFail               = newBooleanEnvVar(false, "SCOPE_TESTING_PANIC_AS_FAIL")
-	ScopeConfiguration                    = newSliceEnvVar([]string{tags.PlatformName, tags.PlatformArchitecture, tags.GoVersion}, "SCOPE_CONFIGURATION")
-	ScopeMetadata                         = newMapEnvVar(nil, "SCOPE_METADATA")
-	ScopeInstrumentationHttpPayloads      = newBooleanEnvVar(false, "SCOPE_INSTRUMENTATION_HTTP_PAYLOADS")
-	ScopeInstrumentationDbStatementValues = newBooleanEnvVar(false, "SCOPE_INSTRUMENTATION_DB_STATEMENT_VALUES")
+	ScopeDsn                                            = newStringEnvVar("", "SCOPE_DSN")
+	ScopeApiKey                                         = newStringEnvVar("", "SCOPE_APIKEY")
+	ScopeApiEndpoint                                    = newStringEnvVar("https://app.scope.dev", "SCOPE_API_ENDPOINT")
+	ScopeService                                        = newStringEnvVar("default", "SCOPE_SERVICE")
+	ScopeRepository                                     = newStringEnvVar("", "SCOPE_REPOSITORY")
+	ScopeCommitSha                                      = newStringEnvVar("", "SCOPE_COMMIT_SHA")
+	ScopeBranch                                         = newStringEnvVar("", "SCOPE_BRANCH")
+	ScopeSourceRoot                                     = newStringEnvVar("", "SCOPE_SOURCE_ROOT")
+	ScopeLoggerRoot                                     = newStringEnvVar("", "SCOPE_LOGGER_ROOT", "SCOPE_LOG_ROOT_PATH")
+	ScopeDebug                                          = newBooleanEnvVar(false, "SCOPE_DEBUG")
+	ScopeTracerGlobal                                   = newBooleanEnvVar(false, "SCOPE_TRACER_GLOBAL", "SCOPE_SET_GLOBAL_TRACER")
+	ScopeTestingMode                                    = newBooleanEnvVar(false, "SCOPE_TESTING_MODE")
+	ScopeTestingFailRetries                             = newIntEnvVar(0, "SCOPE_INSTRUMENTATION_TESTS_FRAMEWORKS_FAIL_RETRIES", "SCOPE_TESTING_FAIL_RETRIES")
+	ScopeTestingPanicAsFail                             = newBooleanEnvVar(false, "SCOPE_INSTRUMENTATION_TESTS_FRAMEWORKS_PANIC_AS_FAIL", "SCOPE_TESTING_PANIC_AS_FAIL")
+	ScopeConfiguration                                  = newSliceEnvVar([]string{tags.PlatformName, tags.PlatformArchitecture, tags.GoVersion}, "SCOPE_CONFIGURATION")
+	ScopeMetadata                                       = newMapEnvVar(nil, "SCOPE_METADATA")
+	ScopeInstrumentationHttpPayloads                    = newBooleanEnvVar(false, "SCOPE_INSTRUMENTATION_HTTP_PAYLOADS")
+	ScopeInstrumentationDbStatementValues               = newBooleanEnvVar(false, "SCOPE_INSTRUMENTATION_DB_STATEMENT_VALUES")
+	ScopeTracerDispatcherHealthcheckFrequency           = newIntEnvVar(60000, "SCOPE_TRACER_DISPATCHER_HEALTHCHECK_FRECUENCY")
+	ScopeTracerDispatcherHealthcheckFrequencyInTestMode = newIntEnvVar(1000, "SCOPE_TRACER_DISPATCHER_HEALTHCHECK_FRECUENCY_IN_TESTMODE")
+	ScopeTracerDispatcherConcurrencyLevel               = newIntEnvVar(1, "SCOPE_TRACER_DISPATCHER_CONCURRENCY_LEVEL")
```

Comment:
By default, we want "1" as concurrency level in Go?

Reply:
unless we decide new defaults for this setting? that is the default in .NET too